modules/crud.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete a single todo
+export const deleteTodo = (id, itemContainer, todoListss) => {
+  const newTodos = todoListss.filter((item) => item.id !== id);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+  if (itemContainer.parentNode) {
+    itemContainer.parentNode.removeChild(itemContainer);
+  }
+};
```

Comment:
- [ ] checking  the `checkbox` of a task should that status of the task of `completed` from `false` to `true`. This should allow the user to delete all selected tasks using the `Clear All Completed` button which is currently no the case. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/214549846-744c60cd-f17f-4020-8c17-6d7de4acef79.png)

Reply:
@Munsa1 I believe the event listener on the checkbox is not a requirement for this milestone, but for the next (third) milestone. However, the student still didn't implement it,  so, I waived it. Thank you 🙏